test_util/azure_test_driver.py

```diff
@@ -62,9 +62,7 @@ def get_value(template_parameter):
 
 
 def get_test_config():
-    # Currently tests only pass with auth disabled, TEST_ADD_ENV_DCOS_AUTH_ENABLED=true
-    # can be set in the environment to override this
-    add_env = {'DCOS_AUTH_ENABLED': 'false'}
+    add_env = {}
```

Comment:
This whole module will be nuked with https://github.com/dcos/dcos/pull/914 so can we just skip the small style changes? Fixing style/idiom inconsistencies in this file is a rabbit hole